docs/content/2.configuration/3.nuxt-auth-handler.md

```diff
@@ -124,6 +124,54 @@ export default NuxtAuthHandler({
 ```
 ::
 
+
+### Cookies
+
+Since the `nuxtAuthHandler` uses `next-auth` in the background, it also creates cookies in order to make the auth mechanisms work.
+There is the option to set a custom name to the cookies if there is the desire or need to do so.
+
+Simply by setting the environment variable `$NUXT_AUTH_COOKIE_NAME`.
```

Comment:
Do you need to do this? As far as I can see in the example below, you can also hardcode it inside the `nuxtAuthHandler.ts`. 

Will is also work "out-of-the-box" by just setting the .env without any other modifications to the `nuxtAuthHandler`?

Reply:
> Do you need to do this? 

no need, but the feature requester @nathanchase requested it and i couldnt think of a reason why not ^^

>Will is also work "out-of-the-box" by just setting the .env without any other modifications to the nuxtAuthHandler?

yes. this is the intention.

<hr/>

idk if its that relevant but we could also link to the OG `next-auth` docs [for this](https://next-auth.js.org/configuration/options#cookies) here..

Reply:
Sounds good. - Then I might add something like "alternatively" to the docs that it is clear that you can also set it in the `nuxtAuthHandler`!

Linking to the NextAuth docs is always very good, as it gives more insight into the underlying logic behind our package!

Reply:
My main concern was the line from the NextAuth docs that said:
_"You can specify one or more cookies with custom properties, but if you specify custom options for a cookie you must provide all the options for that cookie."_

Ideally, we could change just the cookie names, and not have to provide all the other default options.